templates/layout.html.twig

```diff
@@ -83,19 +83,19 @@ Enjoy your theme!
                 <div class="row align-items-center">
                     <div class="col">
                         <div class="page-pretitle">
-                            {% block page_subtitle %}Overview{% endblock %}
+                            {% block page_pretitle %}{% block breadcrumb %}Overview{% endblock %}{% endblock %}
                         </div>
                         <h2 class="page-title">
                             {% block page_title %}Dashboard{% endblock %}
                         </h2>
-                        {% block page_stats %}
+                        {% block page_stats %}{% block page_subtitle %}
```

Comment:
At the end, `page_stats` is no longer needed. `page_subtitle` is enough.

@kevinpapst Do you want to keep it like that or do we remove `page_stats` ?

Reply:
Yeah, if we do a BC break, then fully: you can remove the block completely.